server/app/services/settings/SettingsManifest.java

```diff
@@ -1632,6 +1637,12 @@ public boolean getNorthStarApplicantUi(RequestHeader request) {
                               /* isRequired= */ false,
                               SettingType.STRING,
                               SettingMode.HIDDEN),
+                          SettingDescription.create(
+                              "AZURE_STORAGE_ACCOUNT_PUBLIC_CONTAINER",
```

Comment:
**suggestion:** Suffix with _NAME (but this will get auto updated from the env-var-docs after bin/fmt is run.

Reply:
done